test/test_autograd.py

```diff
@@ -7636,6 +7636,9 @@ def test_non_differentiable(self):
             # No differentiable outputs, shouldn't error
             eq = foo == bar
 
+            # Inplace
+            foo.eq_(bar)
```

Comment:
This is doesn't belong here actually, but I'm not really sure where to put this test either.